package.json

```diff
@@ -80,8 +80,10 @@
     },
     "devDependencies": {
         "ava": "^0.20.0",
+        "babel-cli": "^6.24.1",
         "babel-plugin-transform-flow-strip-types": "^6.22.0",
         "babel-preset-env": "^1.6.0",
+        "babel-preset-flow": "^6.23.0",
```

Comment:
I’m not sure what this is by there’s a babel-plugin above to strip flow using babel.